app/src/androidTest/java/com/github/se/cyrcle/ui/profile/ViewProfileScreenTest.kt

```diff
@@ -415,4 +426,78 @@ class ViewProfileScreenTest {
 
     verify(mockNavigationActions).navigateTo(Screen.PARKING_DETAILS)
   }
+
+  @Test
+  fun whenNoReviews_showsEmptyMessage() {
+    composeTestRule.waitForIdle()
+
+    // Scroll to the reviews section using swipe
+    composeTestRule.onNodeWithTag("ProfileContentSections").performTouchInput { swipeUp() }
+
+    composeTestRule.onNodeWithTag("UserReviewsTitle").assertIsDisplayed()
+
+    composeTestRule
+        .onNodeWithTag("NoReviewsMessage")
+        .assertIsDisplayed()
+        .assertTextEquals("You haven't written any reviews yet")
+  }
+
+  @Test
+  fun whenUserHasReviews_showsReviewsList() {
+    // Add some test reviews
+    val testReview1 =
```

Comment:
testReview instances are already part of this project and have been used everywhere thus far. Please change these tests to use the instances we've created. This is blocking for me as it could ruin the portability of our tests